src/modules/dashboard/components/Dialogs/ManageWhitelistDialog/ManageWhitelistDialogForm.tsx

```diff
@@ -187,7 +197,8 @@ const ManageWhitelistDialogForm = ({
           text={{ id: 'button.confirm' }}
```

Comment:
The colour of the confirm button is dependant on the tab. Add whitelist = green, manage whitelist = pink.
<img width="323" alt="Screenshot 2022-03-28 at 13 33 20" src="https://user-images.githubusercontent.com/582700/160341381-daea62cd-c2e3-4b7d-a235-28c6321944ef.png">
<img width="322" alt="Screenshot 2022-03-28 at 13 33 08" src="https://user-images.githubusercontent.com/582700/160341396-89144ca2-3c92-4c3c-b54c-aa7541b6ad85.png">

Reply:
It's not part of this PR, but I've fixed it now